Symfony/CS/Fixer/PSR2/BracesFixer.php

```diff
@@ -605,4 +606,27 @@ private function getFinalControlContinuationTokensForOpeningToken($openingTokenK
 
         return array();
     }
+
+    private function fixDeclareStrictTypes(Tokens $tokens)
+    {
+        $declares = $tokens->findGivenKind(T_DECLARE);
+
+        foreach ($declares as $index => $declare) {
+            if ($this->isStrictTypeDeclare($index, $tokens)) {
```

Comment:
examples in PHP documentation doesn't have that space, but there is not such a case mentioned here: https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md